app/controllers/foreman_wreckingball/hosts_controller.rb

```diff
@@ -42,6 +42,41 @@ def status_dashboard
       end
     end
 
+    def status_managed_hosts_dashboard
+      @hosts = Host::Managed.authorized(:view_hosts, Host)
+                   .try { |query| params[:owned_only] ? query.owned_by_current_user_or_group_with_current_user : query }
+
+      compute_resources = ComputeResource.where(:type => 'Foreman::Model::Vmware')
+
+      # get all vms by compute resource id
+      vms_by_compute_resource_id = {}
+      # NOTE The call to ComputeResource#vms may slow things down
+      compute_resources.each { |cr| vms_by_compute_resource_id[cr.id] = cr.vms }
```

Comment:
```suggestion
      compute_resources.each { |cr| vms_by_compute_resource_id[cr.id] = cr.vms(eager_loading: true) }
```

This needs to be because:
https://github.com/theforeman/foreman/blob/f146bcbd67be3ef235bf4936916f1d1c0a70858a/app/models/compute_resources/foreman/model/vmware.rb#L53-L62
which leads to
```
NoMethodError (undefined method `uuid' for #<FogExtensions::Vsphere::MiniServers:0x00007ff3da2d1a10>)
```